docs/tables/mongodbatlas_custom_db_role.md

```diff
@@ -0,0 +1,40 @@
+# Table: mongodbatlas_custom_db_role
+
+Custom roles supports a subset of MongoDB privilege actions. These are defined at the project level, for all clusters in the project.
+
+Using custom MongoDB enables you to specify custom sets of actions which cannot be described by the built-in Atlas database user privileges.
+
+## Examples
+
+### Basic Info
+
+```sql
+select
+  role_name,
+  actions
+from
+  mongodbatlas_custom_db_role
+```
+
+### List roles which have the 'FIND' action defined
+
+```sql
+select
+  *
+from
+  mongodbatlas_custom_db_role t,
```

Comment:
```suggestion
  mongodbatlas_custom_db_role as t,
```

I know `as` isn't required, but other plugin queries use it (and mods too), so we should be consistent

Also, is there a reason we use `t` as the alias here, when the table name is `mongodbatlas_custom_db_role`?